### ISSUE TITLE
Warning: iOS 8.0+ / macOS 10.10+ / tvOS 9.0+ / watchOS 2.0+

### ISSUE THREAD

**Issue by stuffmc** [wtw-software/UTMConversion#12]
Every time the compiler runs, we get this error:

```
The iOS Simulator deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 14.4.99.
```

This is because Xcode 12.4 doesn't support iOS 8. Since the iPhone 4S supports iOS 9, the iPhone 5/5G iOS 10, The 5S iOS 12, The iPad Air 1 iOS 12, I wonder if it wouldn't make sense for the library to start having a minimum Version of iOS 12. Where does Swift 4.2 runs anyways? 

By any means, support at least only iOS 9. You also might want to wait Xcode 12.5 and see if it still supports iOS 9.

**Comment by stuffmc** [comment]
You might wanna check this: https://github.com/mxcl/PromiseKit/pull/1170

**Comment by peterringset** [comment]
I think we'll just bump the minimum iOS version, support for iOS 8 is pretty obscure anyway.

**Comment by peterringset** [comment]
Should be resolved now in version 1.4

**Comment by stuffmc** [comment]
It is. That was quick! Thanks!
